src/index.js

```diff
@@ -6,9 +6,9 @@ import fs from 'fs';
 import {createFilter} from 'rollup-pluginutils';
 import {peerDependencies} from '../package.json';
 
-const builtins = builtinList.reduce((set, id) => set.add(id), new Set());
+const builtins = new Set(builtinList);
 
-const ES6_BROWSER_EMPTY = resolve( __dirname, '../src/empty.js' );
+const ES6_BROWSER_EMPTY = '!node-resolve:empty.js';
```

Comment:
As this is kind of a private module, I would assume it should be prefixed with `\0` instead to avoid other plugins from messing with it, e.g. by inserting code?

Reply:
@lukastaegert fixed!